ci/submit_cython_cache.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+CACHE_File="$HOME/.cache/cython_files.tar"
+rm -rf $CACHE_File
+home_dir=$(pwd)
+ls "$TRAVIS_BUILD_DIR/pandas/"
+tar cf "$CACHE_File" \
+"$TRAVIS_BUILD_DIR"/pandas/{index,algos,lib,tslib,parser,hashtable}.c \
```

Comment:
those are all included in other cython files

issue here is to make sure this is kept up to take
when anything is added / changed